libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -57,13 +61,22 @@ private List<Gpg45Scores.Evidence> deduplicateEvidences(List<VerifiableCredentia
         var deduplicatedEvidences = new HashMap<String, List<Gpg45Scores.Evidence>>();
         for (var vc : vcs) {
             if (vc.getCredential() instanceof IdentityCheckCredential idCheckVc) {
-                var docType = getVcDocumentType(vc);
                 var evidence =
                         getEvidences(
                                 idCheckVc.getEvidence() != null
                                         ? idCheckVc.getEvidence()
                                         : Collections.emptyList());
-                if (isEmpty(docType)) {
+
+                // Not an evidence VC - skip
+                if (isNullOrEmpty(evidence)) {
+                    continue;
+                }
```

Comment:
Previously it would blow up at `if (isNullOrEmpty(existing) || evidence.get(0).compareTo(existing.get(0)) > 0) {` if `evidence` was empty.

It was a race condition because it depended whether the NINO VC or the HMRC KBV VC was processed first - if the HMRC KBV VC was processed first, the condition would short-circuit and it would succeed.